connectors/jira.go

```diff
@@ -20,31 +22,162 @@ func (x Jira) Listen(commandMsgs chan<- models.Message, connector models.Connect
 }
 
 func (x Jira) Command(message models.Message, publishMsgs chan<- models.Message, connector models.Connector) {
+	if strings.HasPrefix(strings.ToLower(message.In.Text), strings.ToLower("jira create")) {
+		createJiraIssue(message, publishMsgs, connector)
+	} else {
+		parseJiraIssue(message, publishMsgs, connector)
+	}
+}
+
+func (x Jira) Publish(connector models.Connector, message models.Message, target string) {
+	return
+}
+
+func (x Jira) Help(connector models.Connector) (help string) {
+	help += "jira - mention a jira ticket and it'll be displayed\n"
```

Comment:
The intent of help has been to show commands you can ask of jane, this is just sort of instructional. Maybe put it in the docks. It would be the reason why they would want to add help.